react_frontend/src/assets/JobCard.css

```diff
@@ -0,0 +1,49 @@
+  
+  
+
+  .job-card{
+    position:fixed;
+    list-style: none;
+    font-family: 'Arial';
+    font-weight: 200; 
+    padding: 20px;
+    border-style:solid;
+    border-width: 1mm;
+    width:50%;
+    left:25%;
+    bottom:25%;
+    color: black;
+    border-color: #6A040F;
+    background-color: #6A040F;
+  }
```

Comment:
LGTM

Reply:
If you wanted to do something about super long words this is probably where it would happen, but I think it is fair to assume that we won't be getting input that is so broken.

Reply:
Ya, I will use that advice when tackling front end issues later.